### ISSUE TITLE
can only concatenate list

### ISSUE THREAD

**Issue by troykelly** [robmarkcole/HASS-Deepstack-object#102]
`image_processing.py` is throwing an error concatenating a list.

v2.6 through HACS

```
2020-02-13 10:41:58 ERROR (MainThread) [homeassistant.core] Error doing job: Task exception was never retrieved
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 284, in async_update_ha_state
    self._async_write_ha_state()
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 330, in _async_write_ha_state
    unit_of_measurement = self.unit_of_measurement
  File "/config/custom_components/deepstack_object/image_processing.py", line 279, in unit_of_measurement
    return target + "s"
TypeError: can only concatenate list (not "str") to list
```

**Comment by robmarkcole** [comment]
Ahh yes this is a bug

**Comment by robmarkcole** [comment]
Fixed in release 2.7
